specification/sql/resource-manager/Microsoft.Sql/preview/2022-08-01-preview/ManagedLedgerDigestUploads.json

```diff
@@ -0,0 +1,335 @@
+          },
+          "202": {
```

Comment:
>202 [](http://example.com/codeflow?start=11&length=3)

As per RPC, you can only use 201 and 200 in Put Response, 202 is not allowed, even for async scenarios.

Reply:
I am checking with SQL Rest API team, if we should use 201 instead of 202. I will revert back to you on this.

Reply:
As per https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#creatingupdating-using-put:
Some of the resource provider's existing PUT APIs return 202 Accepted. Please note this is a old model. For PUT requests, 201 / 200 + ProvisioningState is the preferred model. 

The existing https://github.com/Azure/azure-rest-api-specs/blob/549bc277952e66577d69ce137eea810c9815ef22/specification/sql/resource-manager/Microsoft.Sql/preview/2022-05-01-preview/examples/LedgerDigestUploadsEnable.json is using 202, changing this to 201 would cause more confusion.

Reply:
I would imagine the old model will need to be updated, will not block this for now, but you should have a plan on how to update this to new model.